coalib/parsing/ConfParser.py

```diff
@@ -123,6 +123,12 @@ def __parse_lines(self, lines, origin):
                 if key == '':
                     continue
 
+                if key in current_section.contents and keys != []:
```

Comment:
is it possible to omit `keys != []`? either it's empty (so loop won't entered anyway) or it's `keys` which is then not empty.
As far as I understand^^